stripe/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -47,12 +51,23 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
     protected val _paymentMethods = MutableLiveData<List<PaymentMethod>>()
     internal val paymentMethods: LiveData<List<PaymentMethod>> = _paymentMethods
 
+    /**
+     * Request to retrieve the value from the repository happens when initialize any fragment
+     * and any fragment will re-update when the result comes back.
+     * Represents what the user last selects (add or buy) on the
+     * [PaymentOptionsActivity]/[PaymentSheetActivity], and saved/restored from the preferences.
+     */
     private val _savedSelection = MutableLiveData<SavedSelection>()
     private val savedSelection: LiveData<SavedSelection> = _savedSelection
 
     private val _transition = MutableLiveData<TransitionTargetType?>(null)
     internal val transition: LiveData<TransitionTargetType?> = _transition
 
+    /**
+     * On [BaseAddCardFragment] this is set every time the add
+     * card is determined to be valid (not necessarily selected)
```

Comment:
What is the add card?

Reply:
This is the add card fragment.
Updated to: 
     * On [BaseAddCardFragment] this is set every time the details in the add
     * card fragment is determined to be valid (not necessarily selected)